annotations/views/rest_views.py

```diff
@@ -477,25 +481,87 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data,
                         status=status.HTTP_201_CREATED,
                         headers=headers)
-
     @list_route()
     def search(self, request, **kwargs):
         q = request.GET.get('search', None)
-        if not q:
-            return Response({'results': []})
-        pos = request.GET.get('pos', None)
-        concepts = goat.Concept.search(q=q, pos=pos, limit=50)
-
-        def _relabel(datum):
-            _fields = {
-                'name': 'label',
-                'id': 'alt_id',
-                'identifier': 'uri'
-            }
-
-            return {_fields.get(k, k): v for k, v in datum.iteritems()}
-        return Response({'results': map(_relabel, [c.data for c in concepts])})
-
+        if q.lower() == 'jean bacon' and cache.get('jean bacon') != None:
+            results = cache.get('jean bacon')
+            print "success"
+        else:
```

Comment:
?

Reply:
@jdamerow  Are you confused about the whole thing or just me leaving the print statement?

Reply:
yes the whole thing. looks like code to check how the cache works. what would be the point in only checking for jean bacon in the cache?

Reply:
I only cache the jean bacon query. I check to see if jean bacon is queried and then check to make sure it exists in the cache.

Reply:
that doesn't make much sense. why only cache one single thing if you can cache whatever is searched? just use the search string instead of the hard-coded string and it's universally applicable.

Reply:
just cache, what you're caching right now (all results?) and then we worry about caching concepts in general in another ticket.